auth/services/contract/notary.go

```diff
@@ -214,6 +213,13 @@ func (n *notary) Configure() (err error) {
 		n.verifiers[uzi.VerifiablePresentationType] = uziVerifier
 	}
 
+	if _, ok := cvMap[selfsigned.ContractFormat]; ok {
+		ss := selfsigned.NewSessionStore(n.vcr)
```

Comment:
my spouse would be offended by this short var name (well maybe not, but I don't like these short names).

If you don't like writing you could write;

```
n.verifiers[selfsigned.VerifiablePresentationType] = n.signers[selfsigned.ContractFormat] = selfsigned.NewSessionStore(n.vcr)
```

Reply:
I don't like long lines of code.